examples/rn-connection-and-error/README.md

```diff
@@ -204,19 +210,46 @@ To sync data used in this app you must first:
 
 > If you set up your App Services App [via a CLI](#via-a-cli-recommended), you can **skip this step** as these functions should already be defined for you.
 
-We will add a function for forcing a client reset. The function is solely used for demo purposes and should not be used in production. We will also add a function to be run on user creation that adds fields to the user's [custom user data](https://www.mongodb.com/docs/atlas/app-services/users/custom-metadata/) document.
+We will add functions for the following:
+  * Forcing a client reset.
+    * This function is solely used for demo purposes and should not be used in production.
+  * Setting the default `storeId` for a user on creation on the associated [custom user data](https://www.mongodb.com/docs/atlas/app-services/users/custom-metadata/) document.
+    * An [authentication trigger](https://www.mongodb.com/docs/atlas/app-services/triggers/authentication-triggers/) will need to be created and configured to call this function.
+  * Switching the associated Store for the current user.
+  * Creating a new store document and getting all store documents.
+    * These are used to check if the demo stores already exist and create them if not.
```

Comment:
```suggestion
    * These documents are used to check if the demo stores already exist and create them if not.
```

Reply:
I actually I'm referring to the "methods"